codeql/windows-drivers/queries/suites/windows_driver_recommended.qls

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT license.
 
 - description: Security queries recommended for Windows Drivers
```

Comment:
Since this is the recommended file (which includes the MustFix queries as an import, plus the recommended ones), how does "recommended and required queries for Windows Drivers"sound?